query-data/dashboards.mdx

```diff
@@ -29,13 +29,15 @@ Select a private or shared dashboard from this tab. A private dashboard is one t
 
     You see an empty dashboard.
 
-2. Click **+ Add a chart** in the top right corner.
+1. Click **+ Add a chart** in the top right corner.
 
     <Frame caption="New chart button">
         <img src="/doc-assets/shots/your-first-query-1.png" alt="New chart button" />
```

Comment:
IMO it's better to train users to use the "+" button in the top right corner from the start. The other "add a chart" card is only visible in new dashboards without charts, and we will probably revise the empty dashboard exprience very soon.

Reply:
True that. Updated along with some other things.